pkg/core/system.go

```diff
@@ -291,6 +294,17 @@ func deleteNamespaces(kc *kubectlClient, namespaces []string) error {
 	return nil
 }
 
+func deleteNamespaceResource(kc *kubectlClient, namespace string, resourceType string, name string) error {
+	fmt.Printf("Deleting %s/%s resource in %s\n", resourceType, name, namespace)
+	deleteLog, err := kc.kubeCtl.Exec([]string{"delete", "-n", namespace, resourceType, name})
+	if err != nil {
+		if !strings.Contains(deleteLog, "NotFound") {
```

Comment:
Wouldn't it be more robust to use `errors.IsNotFound(err)` (see [this example](https://github.com/projectriff/riff/blob/10227fb7a68677476415ee290710c9cea00a0b34/pkg/core/namespace.go#L89)) instead of a string search?

Reply:
Probably worth making that explicit by factoring out a common function (e.g. `isKubectlNotFound(error) bool`) and calling it from both places.